AdSecCore/Functions/ResultFunction.cs

```diff
@@ -164,7 +164,7 @@ public override void UpdateOutputParameter() {
       MomentRangesOutput.Description = UncrackedMomentRangesOutput.Description;
     }
 
-    protected virtual bool ValidateInputs() {
+    protected virtual bool ValidateSolutionInputs() {
```

Comment:
maintainability: I am just seeing this. Should we not have a generic function that checks that if an input is not optional and its value is null, then we add a warning or error?